bufferpool/bufpool_test.go

```diff
@@ -14,14 +14,21 @@
 package bufferpool
 
 import (
-	"github.com/stretchr/testify/assert"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
 )
 
 func TestBufferPool(t *testing.T) {
+	v := "do be do be do"
+
 	buff := GetBuffer()
-	buff.WriteString("do be do be do")
-	assert.Equal(t, "do be do be do", buff.String())
+	_, err := buff.WriteString(v)
```

Comment:
WriteString will never return an error; and if it had ever returned an error, the content assertion would faill.